basic-features.md

```diff
@@ -4,77 +4,127 @@ summary: Learn about the basic features of TiDB.
+| [JSON functions](/functions-and-operators/json-functions.md)                         | Experimental | Experimental | Experimental | Experimental | Experimental |
+| [Aggregation functions](/functions-and-operators/aggregate-group-by-functions.md)    | Y     | Y   | Y   | Y   | Y   |
+| [Window functions](/functions-and-operators/window-functions.md)                     | Y     | Y   | Y   | Y   | Y   |
+| [Miscellaneous functions](/functions-and-operators/miscellaneous-functions.md)       | Y     | Y   | Y   | Y   | Y   |
+| [Operators](/functions-and-operators/operators.md)                                   | Y     | Y   | Y   | Y   | Y   |
+| **Character sets**                                                                   | **5.0** | **4.0** | **3.1** | **3.0** | **2.1** |
+| [`utf8`](/character-set-and-collation.md)                                            | Y     | Y   | Y   | Y   | Y   |
+| [`utf8mb4`](/character-set-and-collation.md)                                         | Y     | Y   | Y   | Y   | Y   |
+| [`ascii`](/character-set-and-collation.md)[1]                                        | Y     | Y   | Y   | Y   | Y   |
+| [`latin1`](/character-set-and-collation.md)                                          | Y     | Y   | Y   | Y   | Y   |
+| [`binary`](/character-set-and-collation.md)                                          | Y     | Y   | Y   | Y   | Y   |
+| **Collations**                                                                       | **5.0** | **4.0** | **3.1** | **3.0** | **2.1** |
+| [`utf8_bin`](/character-set-and-collation.md)                                        | Y     | Y   | Y   | Y   | Y   |
+| [`utf8_general_ci`](/character-set-and-collation.md)                                 | Experimental | Experimental | N   | N   | N   |
+| [`utf8_unicode_ci`](/character-set-and-collation.md)                                 | Experimental | Experimental | N   | N   | N   |
+| [`utf8mb4_bin`](/character-set-and-collation.md)                                     | Y     | Y   | Y   | Y   | Y   |
+| [`utf8mb4_general_ci`](/character-set-and-collation.md)                              | Experimental     | Experimental   | N   | N   | N  |
+| [`utf8mb4_unicode_ci`](/character-set-and-collation.md)                              | Experimental     | Experimental   | N   | N   | N  |
+| [`ascii_bin`](/character-set-and-collation.md)                                       | Y     | Y   | Y   | Y   | Y   |
+| [`latin1_bin`](/character-set-and-collation.md)                                      | Y     | Y   | Y   | Y   | Y   |
+| [`binary`](/character-set-and-collation.md)                                          | Y     | Y   | Y   | Y   | Y   |
+| **Indexing and constraints**                                                         | **5.0** | **4.0** | **3.1** | **3.0** | **2.1** |
+| Expression Indexes                                                                   | Experimental | Experimental | N   | N   | N   |
```

Comment:
we can use this link for expression index: https://docs.pingcap.com/tidb/stable/sql-statement-create-index#expression-index